sidecar/proxy/envoy/manager.go

```diff
@@ -312,6 +313,20 @@ func buildClusters(rules []api.Rule) []Cluster {
 			ConnectTimeoutMs: 1000,
 		}
 
+		if backend.Timeout > 0 {
+			// convert from float to int
+			cluster.ConnectTimeoutMs = int(backend.Timeout * 1000)
+		}
```

Comment:
That will mean we have to do the inverse conversion in NGINX. Do we care about keeping NGINX up to date?